src/openrct2/paint/paint.c

```diff
@@ -46,13 +46,13 @@ static uint32 _paintQuadrantFrontIndex;
 static paint_struct *_paintQuadrants[MAX_PAINT_QUADRANTS];
 void *g_currently_drawn_item;
 paint_entry * gEndOfPaintStructArray;
-sint16 gUnk9DE568;
-sint16 gUnk9DE56C;
+rct_xy16 gPaintSpritePosition;
+// sint16 gUnk9DE568;
```

Comment:
Probably better to just remove this, rather than commenting out.

Reply:
Ops, that was not my intention.